Corona-Warn-App/src/main/res/values/antigen_strings.xml

```diff
@@ -0,0 +1,88 @@
+    <string name="ag_homescreen_card_status_result_available">"Testergebnis abrufen"</string>
+    <!-- XTXT: Homescreen card: status subtitle - error -->
+    <string name="ag_homescreen_card_status_error">"Fehlerhafter Test"</string>
+    <!-- XTXT: Homescreen card: status subtitle - invalid -->
+    <string name="ag_homescreen_card_status_invalid">"Nicht mehr gültig"</string>
+    <!-- XTXT: Homescreen card: status subtitle - findings -->
+    <string name="ag_homescreen_card_status_findings">"Befund"</string>
+    <!-- XTXT: Homescreen card: subtitle - corona official name -->
+    <string name="ag_homescreen_card_status_name_of_the_cause_of_this_app">"SARS-CoV-2"</string>
+    <!-- XTXT: Homescreen card: status - negative -->
+    <string name="ag_homescreen_card_status_negative">"Negativ"</string>
+    <!-- XTXT: Homescreen card: status - positiv -->
+    <string name="ag_homescreen_card_status_positiv">"Positiv"</string>
+
+    <!-- ####################################
+       Homescreen cards - common body
+   ###################################### -->
+    <!-- XTXT: homescreen card: body - result available -->
+    <string name="ag_homescreen_card_body_result_available">"Warnen Sie Ihre Mitmenschen, wenn Sie positiv auf Corona getestet wurden."</string>
+    <!-- XTXT: homescreen card: body - error -->
+    <string name="ag_homescreen_card_body_error">"Ihr Test konnte nicht ausgewertet werden."</string>
+    <!-- XTXT: homescreen card: body - not valid test -->
+    <string name="ag_homescreen_card_body_not_valid_test">"Ihr Test liegt länger als 21 Tage zurück und ist daher nicht länger relevant. Bitte löschen Sie den Test. Danach können Sie einen neuen Test hinzufügen. "</string>
+    <!-- XTXT: homescreen card: body - negative -->
+    <string name="ag_homescreen_card_body_result_negative">"Das Virus SARS-CoV-2 wurde bei Ihnen nicht nachgewiesen."</string>
+    <!-- XTXT: homescreen card: body - positive -->
+    <string name="ag_homescreen_card_body_result_positive">"Das Virus SARS-CoV-2 wurde bei Ihnen nachgewiesen."</string>
+
+    <!-- ####################################
+        Homescreen cards - rapid test
+    ###################################### -->
+    <!-- XHED: rapid test homescreen card: title -->
+    <string name="ag_homescreen_card_rapidtest_title">"Schnelltest"</string>
+    <!-- XBUT: rapid test homescreen card: dont show anymore button -->
+    <string name="ag_homescreen_card_rapidtest_dont_show_anymore_button">"Nicht mehr anzeigen"</string>
+    <!-- XTXT: rapid test homescreen card: status subtitle - outdated test -->
+    <string name="ag_homescreen_card_rapidtest_status_outdated_test">"Test nicht mehr aktuell"</string>
+    <!-- XTXT: rapid test homescreen card: result positive - check results with PCR test -->
+    <string name="ag_homescreen_card_rapidtest_body_result_positive_pcr_check">"Machen Sie einen PCR-Test um dieses Test-Ergebnis zu verifizieren."</string>
```

Comment:
Sorry, could you please add a comma here: Machen Sie einen PCR-Test, um...
Unfortunately the comma was also missing in Figma. I have since corrected this.